### ISSUE TITLE
Cannot update to 10.14.4

### ISSUE THREAD

**Issue by chenyisen** [syscl/XPS9350-macOS#225]
Syscl:
Cannot update to 10.14.4 directory by OTA and even failed with all-new install. Within old version of clover, the process stocked in com.apple.displaypoilcyd exited with anormal code:1. And I also updated clover to version 4910 and cannot load into install progess with automatic reboot. 
Hope for your solution if you have space time.

**Comment by chenyisen** [comment]
The last message shows"datazone_init:467: allocation zone on dev 0 for allocations of 2 block starting at paddr11501560" and my device reboot automatically.

**Comment by chathil** [comment]
same issue here.

**Comment by x4080** [comment]
Do you guys have success go into the mojave boot disk ? I cannot make it boot (take very long time 1 time and no keyboard and touchpad working)

**Comment by chathil** [comment]
yes, it took me like 15minutes, make sure voodoops2controller kext is in the boot drive. if keyboard and touchpad still not working try to use external device.

**Comment by x4080** [comment]
In high sierra, when it takes a long time it means there's something wrong with the files, anyone that success booting with installer takes a long time too?

@chathil , did you already success installing mojave yet?

**Comment by ramshell2019** [comment]
My success was upgraded to 10.14.4，but this efi is not used.
https://drive.google.com/open?id=1wgPsACdvyPDYPypqn8v7s28A6ZGqimQf
This efi is using FakeSMC.kext.
Use this efi to upgrade to 10.14.4, replace the original efi after the upgrade is complete, remember to upgrade the clover version.
Use the author efi to start slow problems can try to delete VerbStub.kext

**Comment by x4080** [comment]
@ramshell2019 Thanks, I'll try later and let you know

**Comment by chathil** [comment]
it worked, thank you @ramshell2019

**Comment by x4080** [comment]
@chathil do you create boot disk using @ramshell2019 EFI or upgrade from sierra/high sierra?

**Comment by chathil** [comment]
@x4080 i use @ramshell2019  EFI to update to 10.14.4 not upgrade from sierra to mojave.

**Comment by x4080** [comment]
Thanks @chathil
